language-support/java/bindings/src/main/java/com/daml/ledger/javaapi/data/CreatedEvent.java

```diff
@@ -95,7 +103,8 @@ public CreatedEvent(
         agreementText,
         contractKey,
         signatories,
-        observers);
+        observers,
+        Instant.EPOCH);
```

Comment:
hm feels a bit fishy but better than making it an optional. Let's stick with epoch

Reply:
yeah, I pondered on it and decided that Optional is messy. Kept it clean and added a comment over the getter